src/components/Navbar.jsx

```diff
@@ -1,19 +1,22 @@
 import { Link } from 'react-router-dom';
 import './styles/Navbar.css';
+import { FaUser } from 'react-icons/fa6';
 
 const Navbar = () => (
   <nav className="navbar">
     <ul>
       <div className="col-1">
-        <li>Bookstore CMS</li>
+        <li className="title-link">Bookstore CMS</li>
         <li>
-          <Link to="/">Books</Link>
+          <Link to="/" className="subtitle-link">Books</Link>
         </li>
         <li>
-          <Link to="/categories">Categories</Link>
+          <Link to="/categories" className="subtitle-link">Categories</Link>
         </li>
       </div>
```

Comment:
- [ ] When you look closely at the navbar given in the Zeplin design, you notice that the active link (Books) has a different color from the inactive link (Categories). Also try as must as possible to use the same font color as those on the Zeplin design for all the text on your page. See screenshots below and fix the color disparity.
- Zeplin design
![image](https://github.com/iamsjunaid/Bookstore/assets/62849872/811dda70-7ea0-4e61-895d-42693b782c05)

- Your design
![image](https://github.com/iamsjunaid/Bookstore/assets/62849872/e7425545-6929-499f-9c65-e1decdd25ad9)